feat: update ixc2 model to handle video chunks

CamiloInx opened the pull request:
Description
---
* Allow IXC2 model to handle video chunk analysis.

camiloaz commented on vision_agent_tools/models/internlm_xcomposer2.py:
```diff
@@ -76,7 +76,8 @@ def __call__(
         video: VideoNumpy | None = None,
         frames: Frames = MAX_NUMBER_OF_FRAMES,
-    ) -> str:
+        chunk_size: int = 1,
```
let's call it `n_chunks` instead, `chunk size` could be confusing.

CamiloInx replied:
Done! I changed the name

camiloaz reviewed (commented):
Do we have tests for this? If so, it should be failing for the image case because now the response is a list with a single item. Can we also add a test for chunks > 1?

CamiloInx commented:
> Do we have tests for this? If so, it should be failing for the image case because now the response is a list with a single item. Can we also add a test for chunks > 1?

@camiloaz No, the models don't have tests incorporated yet on vision-agent-tools. We do tests it on the other repositories as integration tests.

CamiloInx commented:
> > > Do we have tests for this? If so, it should be failing for the image case because now the response is a list with a single item. Can we also add a test for chunks > 1?
> > 
> > 
> > @camiloaz No, the models don't have tests incorporated yet on vision-agent-tools. We do tests it on the other repositories as integration tests.
> 
> We do have tests:
> 
> https://github.com/landing-ai/vision-agent-tools/blob/4f856bd991009616ef2e4c55e9a41872b28f2ec5/tests/tools/test_internlm_xcomposer2.py#L5-L25

Yeah, sorry for that. I forgot that we moved folder where we used to store the tests. Added the corresponding unit test.

camiloaz commented on tests/models/test_internlm_xcomposer2.py:
```diff
@@ -13,6 +13,17 @@ def test_successful_internlm_xcomposer2_for_video(random_video_generator):
+
+    answer = run_inference(video=video_np, prompt=prompt, n_chunks=2)
+
+    assert len(answer) > 1
```
This should be == 2, right?

CamiloInx replied:
Yes, let me fix it
